hub/src/PeerController.cpp

```diff
@@ -39,6 +39,72 @@ void CPeerController::refresh() {
     });
 }
 
+const QString& CPeerController::status_description(const QString &status){
+    static std::map<QString, QString> dct_desp = {
+
+        {"aborted", tr ("The Peer is in an aborted state. This means that it was abruptly "
+                        "stopped without properly closing the session. Run `start` "
+                        "to resume this virtual machine. If any problems persist, you may "
+                        "have to destroy and restart the virtual machine.")},
+
+        {"gurumeditation", tr ("The Peer is in the \"guru meditation\" state. This is a rare case which means "
+                               "that an internal error in VirtualBox caused the Peer to fail. This is always "
+                               "the sign of a bug in VirtualBox. You can try to bring your VM back online "
+                               "with a `start`.")},
+
+        {"running", tr ("The Peer is running. To stop this Peer, you can run `stop` to "
+                        "shut it down.")},
+
+        {"poweroff", tr ("The Peer is powered off. To restart the Peer, simply run 'start'")},
+
+        {"broken", tr ("The VM for the Peer is not created yet. Please wait installation process "
+                       "if you have started.")},
```

Comment:
@Erkin97 "The VM for the Peer is not created yet. If you have started, please wait for installation process."